config/exercise_readme.go.tmpl

```diff
@@ -7,12 +7,23 @@
 For installation and learning resources, refer to the
 [Groovy resources page](https://exercism.io/tracks/groovy/resources).
 
-Run the tests by executing the test script.
+## Setup
+
+Go through the setup instructions for Groovy to install the necessary
+dependencies:
+
+[https://exercism.io/tracks/groovy/installation](https://exercism.io/tracks/groovy/installation)
+
+## Making the test suite pass
+
+Execute the tests with:
 
 ```sh
-$ gradle test
+$ gradlew test
```

Comment:
Shouldn't it be `./gradlew test` ( `gradlew test` fails with "Command 'gradlew' not found")?